README_en.md

```diff
@@ -2,152 +2,313 @@
-| Hand-painted stick figures | class_num:18<br/>train/val:1007/432        | 0.8785        | 0.9107        |
-| Leaves  | class_num:6<br/>train/val:5256/2278        | 0.8212        | 0.8385        |
-| Container vehicle | class_num:115<br/>train/val:4879/2094       | 0.623         | 0.9524        |
-| Chair    | class_num:5<br/>train/val:169/78         | 0.8557        | 0.9077        |
-| Geology    | class_num:4<br/>train/val:671/296         | 0.5719        | 0.6781        |
-
-
-The 100,000 categories' pretrained model can be downloaded here: [download link](https://paddle-imagenet-models-name.bj.bcebos.com/ResNet50_vd_10w_pretrained.tar). More details can be seen in [**Transfer learning tutorial**](https://paddleclas-en.readthedocs.io/en/latest/application/transfer_learning_en.html).
-
-
-### Object detection
-
-In recent years, object detection tasks attract a lot of attention in academia and industry. The ImageNet classification model is often used for pretrained model in object detection, which can directly affect the effect of object detection. Based on 82.39% ResNet50_vd pretrained model, PaddleDetection provides a Practical Server-side Detection solution, PSS-DET. The solution contains many strategies that can effectively improve the performance while taking limited extra computation cost, such as model pruning, better pretrained model, deformable convolution, cascade rcnn, autoaugment, libra sampling and multi-scale training. Compared with the 79.12% ImageNet1k pretrained model, the 82.39% model can help improve the COCO mAP by 1.5% without any computation cost. Using PSS-DET, the inference speed on single V100 GPU can reach 20FPS when COCO mAP is 47.8%, and reach 61FPS when COCO mAP is 41.6%. For more details, please refer to [**Object Detection tutorial**](https://paddleclas-en.readthedocs.io/en/latest/application/object_detection_en.html).
 
+## Features
 
-- TODO
-- [ ] Application of PaddleClas in OCR tasks.
-- [ ] Application of PaddleClas in face detection and recognition tasks.
+- Rich model zoo. Based on the ImageNet1k classification dataset, PaddleClas provides 24 series of classification network structures and training configurations, 122 models' pretrained weights and their evaluation metrics.
 
+- SSLD Knowledge Distillation. Based on this SSLD distillation strategy, the accuracy of the distillation model is generally increased by more than 3%.
 
+- Data augmentation: PaddleClas provides detailed introduction of 8 data augmentation algorithms such as AutoAugment, Cutout, Cutmix, code reproduction and effect evaluation in a unified experimental environment.
 
-## Industrial-grade application deployment tools
+- Pretrained model with 100,000 categories: Baidu self-developed and open sourced the `ResNet50_vd` model trained on a 100,000-category dataset. In some practical scenarios, the  accuracy based on the pretrained weights can be increased by up to 30%.
```

Comment:
"Baidu self-developed and open sourced the `ResNet50_vd` model trained on a 100,000-category dataset"?
The `ResNet50_vd` model Baidu self-developed and open sourced trained on a 100,000-category dataset.

Reply:
fix, thx